doc/news/changes/minor/20160214MathiasMentler

```diff
@@ -0,0 +1,6 @@
+Fixed: double contraction of two SymmetricTensors<..,VectorizedArray<T>> now
```

Comment:
If you spell it `SymmetricTensor` (without the plural), then doxygen will pick this up correctly.

While there, capitalize "double" at the beginning of the sentence...